con-base/src/test/java/ba/codecentric/base/validation/FacilityTest.java

```diff
@@ -0,0 +1,149 @@
+package ba.codecentric.base.validation;
+
+import ba.codecentric.base.domain.Facility;
+import ba.codecentric.base.domain.FacilityType;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.boot.SpringBootConfiguration;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import javax.validation.ConstraintViolation;
+import javax.validation.Validation;
+import javax.validation.Validator;
+import javax.validation.ValidatorFactory;
+import java.util.Set;
+
+import static org.junit.Assert.assertEquals;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@SpringBootConfiguration
```

Comment:
Do we need these annotations? If I am right in our tests we are not using any custom configurations which would be needed for testing.